### ISSUE TITLE
Response type may be "Unknown" 

### ISSUE THREAD

**Issue by greensweater** [Sage/SDataJavaScriptClientLib#2]
Using this with SalesLogix 7.5.4 and IE8.. We saw a Javascript error in sdata-client.js which was "object doesn't support this property or method" so I put the sdata-client-debug.js library in place and checked where the error was coming from. Looks like line 1816 from SLX, or line 882 here.

var contentType = response.getResponseHeader && response.getResponseHeader('Content-Type');

Turns out the null-check on response.getResponseHeader is the culprit. Seems like perfectly fine syntax, so I did some googling and found this article:

http://www.mail-archive.com/swfobject@googlegroups.com/msg02416.html

Basically, XMLHTTP in IE uses a COM+ bridge so objects may return "unknown" type, resulting in the above error. I changed it to this:

var contentType;
if (typeof(response.getResponseHeader) != 'undefined')
    contentType = response.getResponseHeader('Content-Type');

Seems to do the trick.
